worker/src/response.rs

```diff
@@ -472,3 +472,35 @@ impl From<web_sys::Response> for Response {
         }
     }
 }
+
+/// A trait used to represent any viable Response struct that can be used in the Worker.
+/// The only requirement is that it be convertable to a web_sys::Response.
+pub trait IntoResponse {
+    fn into_raw(self) -> web_sys::Response;
+}
+
+impl IntoResponse for web_sys::Response {
+    fn into_raw(self) -> web_sys::Response {
+        self
+    }
+}
+
+impl IntoResponse for Response {
+    fn into_raw(self) -> web_sys::Response {
+        self.into()
+    }
+}
+
+#[cfg(feature = "http")]
+impl IntoResponse for crate::HttpResponse {
```

Comment:
Can this be more generic?
```suggestion
impl IntoResponse for http::Response<http_body::Body> {
```

Reply:
@kflansburg made more generic in https://github.com/cloudflare/workers-rs/pull/530/commits/370af17e28b7e7065c10f9cdf6813d25d87ed227 (as far as `crate::http::response::to_wasm()` allows: `impl http_body::Body<Data = Bytes> + 'static`)

These seems to cover/conflict with the concrete axum impl too (so I removed that) :)

Reply:
> What's the thought on `impl` for the concrete types rather than the alias?

I don't understand the question. `http_body::Body` is a trait and will be more generic than the type alias.

Reply:
Oh you're right, I just read that as `http::body::Body` which isn't a trait.